### PR TITLE
Adding basic SELECT statements

### PR DESCRIPTION
**Pull request by Lotes**
Includes parser + linker for following cases:
- [X] `SELECT * FROM tab;`
- [X] `SELECT id, name FROM tab;`
- [X] `SELECT t.id, t.name FROM tab t;`
- [x] `SELECT t.id, s.name FROM tab t, tab s;`

### CODE REVIEW

File: src/language-server/sql-meta.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+export type SqlTypeDefinitions = Record<string, SqlTypeDefinition>;
```
Comment (spoenemann) [inline src/language-server/sql-meta.ts:1]: Please add copyright header to new files.

File: package.json [hunk 2]
```diff
@@ -54,15 +65,16 @@
         "@typescript-eslint/parser": "^5.44.0",
         "eslint": "^8.28.0",
         "langium-cli": "0.6.0-next.62c127c",
```
Comment (spoenemann) [inline package.json:67]: Please update langium and langium-cli to 1.0.0

Comment (Lotes) [reply package.json]: Done

File: src/language-server/sql-meta.ts (hunk 1 above)
Comment (Lotes) [reply src/language-server/sql-meta.ts:6]: Done

**Pull request merged**